src/test/java/edu/kis/powp/jobs2d/commands/SubscribeCommandsCounterVisitor.java

```diff
@@ -8,20 +8,19 @@
 import java.util.logging.Logger;
 
 public class SubscribeCommandsCounterVisitor implements Subscriber {
-	DriverCommandManager manager = CommandsFeature.getDriverCommandManager();
```

Comment:
Now you should be able to remove line:
import edu.kis.powp.jobs2d.features.CommandsFeature;
to finalize dependency removal